README.md

```diff
@@ -111,7 +101,15 @@ _**NOTE:** If opting out of one or both of these plugins, **remove** the desired
 
 5. Copy the `.env.sample` into a new `.env` file, and update any necessary `environment` variables in `docker-compose.yml`
 
-6. Run the install script
+6. In `package.json` and `composer.json`, update repository and author information
+
+7. Run the theme command and follow the prompt, which will set up the project with your desired theme name
+
+   ```shell
+   ./bin/theme
```

Comment:
I know it might break our streak of one-word scripts but ... should we be more description with the script name (e.g. `rename-theme`)?

Reply:
I was thinking the same thing 😅 I'll rename it